### DISCUSSION TITLE
Is `flutter_stream_chat` still maintained and when will it be released?

### DISCUSSION THREAD

**Discussion by ValentinVignal** [GetStream/stream-feed-flutter#229] [General]
In pub.dev, [`stream_feed_flutter`](https://pub.dev/packages/stream_feed_flutter) is still at the version `0.0.0` and there is nothing in it. 

In the `README.md`, it is written:
> We are doing some polishing before releasing the UI kit.

I tried to install the pub dependency via git as it is written but it didn't work in my case (see https://github.com/GetStream/stream-feed-flutter/issues/226)

The version `0.0.0` has been released 1 year ago already.

And if I look at the repo activity, it looks like you stopped working on it:

<img width="773" alt="image" src="https://user-images.githubusercontent.com/32538273/170433661-a6f1bb71-d6d8-446a-8080-72302c19f3f8.png">

Is this repository still maintained? If yes, is there a date we can expect the polished version to be released?

**Comment by sachaarbonel** [comment]
We adopted a different strategy for feeds given its generic nature (a Twitter clone is different than a Reddit or Youtube clone etc), it needs more customization than chat so we are pushing towards the core architecture and focusing on DevRel content. Regarding the repo activity, we started doing squash commits so what was a PR with 100 commits are now 1 commit. Check out the core example on pub or the [Instagram clone](https://github.com/GetStream/flutter-instagram-clone) to see how flexible the core SDK is.
